components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilderFactory.java

```diff
@@ -45,6 +45,9 @@
 
     private static final Log LOG = LogFactory.getLog(JsGraphBuilderFactory.class);
     private static final String JS_BINDING_CURRENT_CONTEXT = "JS_BINDING_CURRENT_CONTEXT";
+
+    // Suppress the Nashorn deprecation warnings in jdk 11
+    @SuppressWarnings("removal")
```

Comment:
We need to create a new git issue, try to use GraalVM instead of nashorn, and mention here